admission/hooks/validate_service_test.go

```diff
@@ -0,0 +1,60 @@
+package hooks
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+)
+
+var _ = Describe("test for allowing rules for Service", func() {
+	It("should allow creating Service with externalIPs empty", func() {
+		svc := &corev1.Service{}
+		svc.Name = "allow1"
+		svc.Namespace = "default"
+		svc.Spec.Type = corev1.ServiceTypeClusterIP
```

Comment:
You should test with other types as well, see https://github.com/kubernetes/kubernetes/issues/97076#issuecomment-740317872 (the validation is ok for me)

Reply:
Thank you for telling me that.
I already understood that any other service type than ClusterIP should be validated out,
but the logic of the webhook is so simple that we just left the test as it is now.